pkg/rhttp/datatx/manager/simple/simple.go

```diff
@@ -40,6 +42,18 @@ import (
 	"github.com/cs3org/reva/v2/pkg/utils"
 )
 
+// Keep track of active uploads/downloads.
```

Comment:
this is the 'simple' upload, pleas also check the spaces and tus datatx managers